### DISCUSSION TITLE
RefindPlus Hangs on iMac8,1 (Early 2008)

### DISCUSSION THREAD

**Discussion by pauleec** [dakanji/RefindPlus#191] [General]
I've been trying to use the refindplus efi to help the system load Microsoft UEFI 2.0 efi's. I;m very keen to keep the machine being able to load Windows via the UEFI 2.0 support provided by RefindPlus. Every time the machine loads the efi it locks on the "select boot drive" screen. The log generated by the debug efi always shows the same thing.

[24p01x0541.log](https://github.com/user-attachments/files/16455451/24p01x0541.log)

This is consistent on each reboot to the refindplus efi. The machine is able to boot into the standard refind efi but this does not give the Uefi 2.0 support I need. Can you suggest anything I might be able to do to get this really useful and important enhancement working on the imac8,1.

**Comment by dakanji** [comment]
Please raise an actual issue for this: https://github.com/dakanji/RefindPlus/issues/new?template=bug_report.yml

Thanks
